web_app_drawer/static/src/less/dropdowns.less

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2016 LasLabs Inc.
+ * License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). */
+
+.dropdown-submenu {
```

Comment:
Yes, I noticed that. It's a good option too, if you find them. Otherwise just a bit of CSS overflow fixing on `.navbar-nav` class should do the trick.